homework-submission/todo_react/src/ItemCreate.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import Delete from "./Delete";
+import Edit from "./Edit";
+import DatePicker from 'react-date-picker';
+// import DateP from "./DatePicker";
+
+export default class ItemCreate extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            to_do: [],
+            description: [],
+            deadline: [new Date()]
```

Comment:
I am not sure for your use of an array while you have only a `new Date()` implemented.